gprofiler/metadata/__init__.py

```diff
@@ -0,0 +1,31 @@
+from subprocess import CompletedProcess
```

Comment:
Let's place at `gprofiler/metadata/versions.py`, this needn't be in the package level